packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -338,6 +345,7 @@ contract PerpetualPositionManager is FeePayer, AdministrateeInterface {
         external
         notEmergencyShutdown()
         fees()
+        updateFundingRate()
```

Comment:
Would be curious how much gas this adds to each contract action but can leave for another PR